### DISCUSSION TITLE
Make the FPS Counter a skin element

### DISCUSSION THREAD

**Discussion by rodrigopina360** [ppy/osu#24826] [Ideas] [answered]
By this point I think we have a robust enough skinning system that something like the fps counter would make sense to make available as a skinning element in order to be able to change where it appears.
If people agree with it, I might even be able to implement it myself, as I've very recently messed with a skin editor element, and should be able to know how to create a new one properly

**Comment by rodrigopina360** [comment]
The only issue I foresee with the change is people not understanding that the fps counter it tied to the skin, as they are currently used to it being a single toggle setting, which would also need to be discussed.
Like is the toggle in settings going away? Is it staying and toggling fps counter in the skin if it exists? Etc...

**Comment by peppy** [comment, marked as answer]
I disagree with this. It should be global. Rationale: I don't think users should be displaying this for normal gameplay. Or ever really, if we've done our job properly.

**Reply by rodrigopina360** [reply]
Yeah, that's fair enough. I was targeting more the people that actually like seeing the fps counter (idk there's people that like seeing big numbers on screen) but I do agree that in a perfect would people would just not have it enabled
